coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -849,6 +932,16 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin) {
         addressCountry: this.inProgressAddress.addressCountry,
         deliveryInstructions: this.inProgressAddress.deliveryInstructions
 
+      },
+      mailingAddress: {
```

Comment:
Would it be possible to type `mailingAddress: { ...this.inProgressMailAddress }` instead and greatly clean up this code?

Reply:
100% it is possible, let's use the spread operator where we can, great tip, thank you!